src/Schedule.cpp

```diff
@@ -301,6 +316,14 @@ void Schedule::accept(IRVisitor *visitor) const {
             b.remainder.accept(visitor);
         }
     }
+    for (const Prefetch &p : prefetches()) {
+        debug(4) << "Schedule::accept prefetches\n";
```

Comment:
I actually used debug levels 4 & 5 quite heavily to get the realize node support working.